changedetectionio/tests/test_notification.py

```diff
@@ -68,6 +70,14 @@ def test_check_notification(client, live_server):
     # Give the thread time to pick up the first version
     time.sleep(3)
 
+    testimage = 'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII='
+    # Write the last screenshot png
+    datastore = '/tmp/changedetection.io/changedetectionio/test-datastore/'
+
+    for datastoreelement in glob(datastore + "*/"):
+        with open(datastoreelement + 'last-screenshot.png', 'wb+') as f:
```

Comment:
Why `+` ? no need for append, just `wb`

```suggestion
        with open(datastoreelement + 'last-screenshot.png', 'wb') as f:
```